dhcpv6/option_userclass.go

```diff
@@ -50,5 +50,9 @@ func ParseOptUserClass(data []byte) (*OptUserClass, error) {
 		len := buf.Read16()
 		opt.UserClasses = append(opt.UserClasses, buf.CopyN(int(len)))
 	}
-	return &opt, buf.FinError()
+	var err = buf.FinError()
```

Comment:
Modifying the return error message of the user class option does not seem quite related to the topic of the PR.

While I am personally not against it, this makes it inconsistent with what we log for other options.

Why specifically the user class? Can all these "logging improvements" be discussed in a separate PR and in this one only focus on the ZTP changes?

Reply:
I can move these changes out.

I just added the error messages that helped me troubleshooting some issues I had at the beginning.